### ISSUE TITLE
Content-Type: text/xml not escaped

### ISSUE THREAD

**Issue by MiRacLe-RPZ** [tarampampam/webhook-tester#77]
```
curl -X POST --location "https://web.hook.sh/$guid-here$" \
    -H "Content-Type: text/xml" \
    -d "<hello>world</hello>"

```

Shown as "world" (without tags)

**Comment by tarampampam** [comment]
Bug confirmed, tomorrow will be fixed. Thanks for your report!

**Comment by tarampampam** [comment]
💥 Released in [v0.3.1](https://github.com/tarampampam/webhook-tester/releases/tag/v0.3.1), thank you!
